### ISSUE TITLE
Updating to MaxTeX 2016 broke something

### ISSUE THREAD

**Issue by schar** [semprag/biblatex-sp-unified#21]
Here is a simple document: 

```
\documentclass{article}
\usepackage[%
  backend      = biber,
  bibstyle     = biblatex-sp-unified,
  citestyle    = sp-authoryear-comp,
]{biblatex}
\addbibresource{test.bib}
\begin{document}

\cite{test}

\printbibliography{}
\end{document}
```

Which refers to this simple `bib` file:

```
@article{test,
  Author = {test},
  Title = {test}
}
```

This used to work. After updating my MacTeX distribution this Sunday morning and (full disclosure) doing a `sudo tlmgr update --all`, I now receive errors like the following: 

```
! Undefined control sequence.
<argument> \bbl@main@language

l.12 \printbibliography{
                        }
?
```

Curiously, if either the `Author` or `Title` field is missing, I don't get an error. Moreover, if I remove the `bibstyle` option, the error goes away (but not if I remove the `citestyle` option). This leads me to suspect that the problem is somewhere in `biblatex-sp-unified.bbx`, but that's as far as my limited knowledge and expertise gets me.

**Comment by schar** [comment]
Per [this answer and the highlighted comment in particular](http://tex.stackexchange.com/questions/315532/mactex-2016-somehow-broke-a-bbx/315537?noredirect=1#comment769961_315537), this can be resolved by loading `babel` with a language option (e.g., `english`). See also the discussion [here](https://groups.google.com/forum/#!msg/comp.text.tex/FKahkrwlS4Q/Q6bJgcCjDAAJ).

**Comment by fintelkai** [comment]
@schar Thanks for raising this issue and finding the solution. We knew that BibLaTeX 3.3./3.4 is a bit of a disaster for our style file(s) and trying to figure what else it broke is on our plate.

**Comment by fintelkai** [comment]
I have now read up on the issue. This is a bug in Biblatex v3.4 and is supposed to be fixed on v3.5. So, I suppose we won't change biblatex-sp-unified. On the other hand, I doubt anybody has actually made use of the fact that we allow setting a `langid` field in individual bib-entries, which then is taken account when a title is made into sentence case. If we could live without that future-proofing, we could just use `\MakeSentenceCase` instead of the starred variant (which is what we use and what is sensitive to the `langid` field).
